perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -1200,13 +1237,49 @@ def ListConsistencyBenchmark(storage_schema, host_to_connect=None):
                final_result)
 
   # Final clean up: delete the objects still remaining.
-  DeleteObjectsConcurrently(storage_schema,
+  DeleteObjectsConcurrently(backend,
                             per_thread_objects_written,
-                            host_to_connect,
                             per_thread_objects_deleted)
   return final_result
 
 
+def TestBackend(backend):
```

Comment:
It seems to me that for a test method like this, it shouldn't depend on actual bucket or actual provider. It should be runnable in a tox environment as a unit-test. 

Can you make a fake backend, and call this test using a fake backend? Then you can move this method out of this file to the test directory.